pymemcache/client/base.py

```diff
@@ -941,12 +937,12 @@ def _raise_errors(self, line, name):
 
     def _check_integer(self, value, name):
         """Check that a value is an integer and encode it as a binary string"""
-        if not isinstance(value, six.integer_types):
+        if not isinstance(value, int):
             raise MemcacheIllegalInputError(
-                '%s must be integer, got bad value: %r' % (name, value)
+                f'{name} must be integer, got bad value: {value!r}'
             )
 
-        return six.text_type(value).encode(self.encoding)
+        return str(value).encode(self.encoding)
```

Comment:
Could these cases be further converted to `bytes(value, self.encoding)`?

Reply:
```>>> bytes('5', 'ascii')
b'5'
>>> bytes(5, 'ascii')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: encoding without a string argument
>>> bytes(5, 'utf-8')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: encoding without a string argument```

bytes won't work here since value is an int